### PR TITLE
added 24 hour volume to the Market details Modal

### PR DESCRIPTION
**Pull request by chriscanal**
This is my first contribution, I'm trying something small, to meet you guys and better understand how things work. 24 hour volume is something that I'm curious about often, so I think it would be a nice addition to this modal.

### CODE REVIEW

File: web/components/contract/contract-info-dialog.tsx [hunk 1]
```diff
@@ -142,7 +142,15 @@ export function ContractInfoDialog(props: {
...
-                    <span className="mr-1">Volume</span>
+                    <span className="mr-1">24 Hour Volume</span>
+                    <InfoTooltip text="The amount bought or sold in the last 24 hours" />
+                  </td>
+                  <td>{formatMoney(contract.volume24Hours)}</td>
+                </tr>
+
+                <tr>
+                  <td>
+                    <span className="mr-1">Total Volume</span>
```
Comment (jahooma) [inline web/components/contract/contract-info-dialog.tsx:153]: I'd lowercase the words that are not first, so `24 hour volume` and `Total volume`

**Review by jahooma — APPROVED** [review]
Thanks Chris! Glad to have your first contribution.

**Comment by chriscanal** [comment]
image

File: web/components/contract/contract-info-dialog.tsx (hunk 1 above)
Comment (chriscanal) [reply web/components/contract/contract-info-dialog.tsx]: Updated :)

**Pull request merged**